soccer/gameplay/plays/restarts/basic_indirect.py

```diff
@@ -0,0 +1,53 @@
+import standard_play
+import behavior
+import skills.move
+import skills.pivot_kick
+import constants
+import robocup
+import math
+import main
+import tactics.coordinated_pass
+import evaluation.passing_positioning
+import enum
+
+
+class BasicIndirect(standard_play.StandardPlay):
+
+    def __init__(self, indirect=None):
+        super().__init__(continuous=True)
+
+        self.add_transition(behavior.Behavior.State.start,
+                            behavior.Behavior.State.running, lambda: True,
+                            'immediately')
+
+        # FIXME: this could also be a PivotKick
```

Comment:
I would remove this comment. At the current moment we have decided to use LineKick's for all restarts and pivot kicks for passing and normal gameplay